mozilla/foundation.mozilla.org
# background looping video streamfield

Closes #6981 

**Quick Notes:** 

- At the moment, the wagtail test will fail since I have not added the migrations yet. Once I get approval from reviewers, I will pull the latest migrations from master and then run makemigrations to make sure everything is up to date.
- As I branched off from a existing branch of mine for reference, it seems that it has carried some of the old commits with it. Sorry about that! 


### Testing:
**Link to sample test page**: https://foundation-s-6981-backg-iy4p9d.herokuapp.com/en/blog/initial-test-blog-post-with-fixed-title/

**Steps to test**

1. Visit [the test blog page](https://foundation-s-6981-backg-iy4p9d.herokuapp.com/en/blog/initial-test-blog-post-with-fixed-title/), and scroll down to the full width video. 
2. This video should be responsive, and should also play/pause when you click the button on the right.
3. Scroll down further to find the next looping video block
4. The second video block should also be responsive, play/pause as expected.



## Checklist

**Changes in Models:**
- [x] [Are my changes backward-compatible](https://github.com/mozilla/foundation.mozilla.org/blob/master/docs/workflow.md#django-migrations-what-to-do-when-working-on-backward-incompatible-migrations). If not, did I schedule a deploy with the rest of the team?
